docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,822 @@
+                          iconColor="darkGray"
+                          onClick={() => setAlignText('left')}
+                          size="lg"
+                          selected={alignText === 'left'}
+                        />
+                      </Tooltip>
+                      <Tooltip text="Align center" zIndex={TOOLTIP_Z_INDEX}>
+                        <IconButton
+                          accessibilityLabel="Align center"
+                          bgColor="white"
+                          icon="text-align-center"
+                          iconColor="darkGray"
+                          onClick={() => setAlignText('center')}
+                          size="lg"
+                          selected={alignText === 'center'}
+                        />
+                      </Tooltip>
+                      <Tooltip text="Align right" zIndex={TOOLTIP_Z_INDEX}>
+                        <IconButton
+                          accessibilityLabel="Align right"
+                          bgColor="white"
+                          icon="text-align-right"
+                          iconColor="darkGray"
+                          onClick={() => setAlignText('right')}
+                          size="lg"
+                          selected={alignText === 'right'}
+                        />
+                      </Tooltip>
+                    </Flex>
+                  </Flex>
+              </Box>
+            </ScrollableContainer>
+          </Modal>
+        </Layer>
+      )}
+    </>
+  )
+}`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="With scrollableContainer"
+      description="ScrollableContainer is needed for proper positioning when the Tooltip is anchored to an element that is located within a scrolling container. The use of ScrollableContainer ensures the Tooltip remains attached to its anchor when scrolling."
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+function ScrollableContainerExample() {
+  const [content, setContent] = React.useState(null);
+  const [claimed, setClaimed] = React.useState(null);
+  const [device, setDevice] = React.useState(null);
+
+  return (
+    <ScrollableContainer
+      overflow="scrollY"
+      height={200}
+    >
+      <Flex
+        width={300}
+        direction="column"
+        gap={4}
+      >
+        <Flex
+          direction="column"
+          gap={2}
+        >
+          <Flex
+            alignItems="center"
+            gap={1}
+          >
+            <Text
+              weight="bold"
+              size="lg"
+            >
+              Content type
+            </Text>
+            <Tooltip
+              idealDirection="right"
+              text="See stats about different types of content created by you and/or others on Pinterest. Filter to get more details on your organic (not an ad) and paid (promoted as an ad) content."
+            >
+              <Icon
```

Comment:
These should each be IconButtons as well

Reply:
DONE